src/GitVersionCore/Model/FileLock.cs

```diff
@@ -0,0 +1,25 @@
+using GitVersion.FileLocking;
+using GitVersion.FileLocking.Abstractions;
+using System;
+using System.IO;
+
+namespace GitVersion.Helpers
+{
+    public class FileLock : IFileLock
```

Comment:
Why isn't `FileLock` placed in the `FileLocking` namespace?

Reply:
Missed it, thanks.